api-flask/app/main.py

```diff
@@ -153,135 +181,149 @@ def stats():
 #     """Get all alerts in current table."""
 #     results = alert_db.readall()
 #     return Response(json.dumps(results, cls=AlchemyEncoder), mimetype='application/json')
-@app.route('/kobo/forms', methods=['GET'])
-def get_kobo_forms():
-    """Get all form responses."""
-    begin_datetime, end_datetime = parse_datetime_params()
-    form_responses = get_form_responses(begin_datetime, end_datetime)
-
-    return Response(json.dumps(form_responses), mimetype='application/json')
-
-
-@app.route('/alerts/<id>', methods=['GET'])
-def alert_by_id(id: str = '1'):
-    """Get alert data from DB given id."""
-    try:
-        id = int(id)
-    except ValueError as e:
-        logger.error(f'Failed to fetch alerts: {e}')
-        raise BadRequest('Invalid id')
-
-    alert = alert_db.readone(id)
-    if alert is None:
-        raise NotFound(f'No alert was found with id {id}')
-
-    # secure endpoint with simple email verification
-    if request.args.get('email', '').lower() != alert.email.lower():
-        raise BadRequest('Access denied. Email addresses do not match.')
-
-    if request.args.get('deactivate'):
-        status = alert_db.deactivate(alert)
-        if not status:
-            raise InternalServerError('Failed to deactivate alert')
-
-        return Response(response='Alert successfully deactivated.', status=200)
-
-    return Response(json.dumps(alert, cls=AlchemyEncoder), mimetype='application/json')
-
-
-@app.route('/alerts', methods=['POST'])
-def write_alerts():
-    """Post new alerts."""
-    if not request.is_json:
-        raise InternalServerError('InvalidInput')
-
-    try:
-        data = request.json
-        alert = AlertModel(**data)
-        alert_db.write(alert)
-
-    except rasterio.errors.RasterioError as e:
-        logger.error(e)
-        raise e
-
-    return Response(response='Success', status=200)
 
+@api.route('/kobo/forms')
+class Get_kobo_forms(Resource):
```

Comment:
Let's have class names in CapWords -> https://peps.python.org/pep-0008/#class-names